spec/requests/healthcheck_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+RSpec.describe "Healthcheck" do
+  describe "GET /healthcheck" do
+    context "when the healthchecks pass" do
+      before do
+        populate_default_government_bulk_data
+      end
+
+      it "returns a status of 'ok'" do
+        get healthcheck_path
+        expect(JSON.parse(response.body)["status"]).to eq("ok")
+      end
+    end
```

Comment:
I don't think this request needs to be concerned about what the statuses are - since there is no logic in the controller about what is returned just which tests are run. 

I think you'd be fine to test that the healthcheck does two things:

1) returns a 200 HTTP status
2) returns JSON with a status attribute (doesn't matter what the value is)

Reply:
Nice, I've changed the tests and removed the context block now that it doesn't make sense